elements/lisk-p2p/src/utils.ts

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { hash } from '@liskhq/lisk-cryptography';
+import { isIPv4 } from 'net';
+
+const SECRET_BUFFER_LENGTH = 4;
+const NETWORK_BUFFER_LENGTH = 1;
+const PREFIX_BUFFER_LENGTH = 1;
+const MOD_4 = 4;
+const MOD_64 = 64;
+
+export enum NETWORK {
+	NET_IPV4 = 0,
+	NET_PRIVATE,
+	NET_LOCAL,
+	NET_OTHER,
+}
+
+/* tslint:disable no-magic-numbers */
+export const getByte = (address: string, n: number): number | undefined => {
+	if (n > 3) {
+		return undefined;
+	}
+
+	return parseInt(address.split('.')[n], 10);
```

Comment:
It won't happen in this flow since isIPv4 check occurs before this function is ever called, but I will add it in case a user wants to use it independently.